var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -117,4 +117,12 @@ def cmake_args(self):
         if 'darwin' in spec.architecture:
             options.append('-DCMAKE_MACOSX_RPATH:BOOL=ON')
 
+        if '--run-tests' or '--test' in spec:
+            options.append('-DENABLE_TESTS:BOOL=ON')
```

Comment:
Might want to explicitly disable in an else clause, just to be safe

Reply:
okay will do